src/shogun/util/factory.h

```diff
@@ -285,10 +288,5 @@ namespace shogun
 		SG_REF(result);
 		return result;
 	}
-
-	CMachine* machine(CPipeline* pipeline)
```

Comment:
@vinx13 I removed this as it is not really needed, one can use `machine` (I renamed to `as_machine` in this PR)